health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/CustomHorizontalBarChart.js

```
@@ -0,0 +1,539 @@
+                        if (chartData[index][properties] !== 0) {
+                          topIndex = i;
+                        }
+                        i += 1;
+                      }
+                    }
+                    return (
+                      <Cell
+                        radius={id === topIndex || bars.length === 2 || chartType == "sideBySideBar" ? [5, 5, 0, 0] : undefined}
+                        stroke={(activeIndex === index) & (activeBarId === id) ? "#ccc" : null}
+                        strokeWidth={3}
+                      />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)